updates/26/updates.sql

```diff
@@ -410,3 +410,5 @@ INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0x7237C0B30B13
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0x6125aDCAb2F171BC70cfe2CAeCFeC5509273A86A", "MetaGaming Guild [via ChainPort.io]", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0x6125aDCAb2F171BC70cfe2CAeCFeC5509273A86A", "A", 56, "0x6125aDCAb2F171BC70cfe2CAeCFeC5509273A86A", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0x6125aDCAb2F171BC70cfe2CAeCFeC5509273A86A", "MGG", "metagaming-guild", "mgg", NULL, 1718801298, NULL);
 *
+INSERT INTO assets(identifier, name, type) VALUES("eip155:324/erc20:0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E", "zkSync", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:324/erc20:0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E", "A", 324, "0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:324/erc20:0x5A7d6b2F92C77FAD6CCaBd7EE0624E64907Eaf3E", "ZK", "zksync", "ZKSYNC", NULL, 1716317326, NULL);
```

Comment:
use the same name as in https://github.com/rotki/data/blob/5b8741277fe9521775bcd7d261873690eaec065e/airdrops/index_v2.json#L268C26-L268C32